src/Tickets/Commerce/Gateways/PayPal/Client.php

```diff
@@ -434,9 +434,14 @@ public function create_order( array $units = [] ) {
 	 *
 	 * @return array|null
 	 */
-	public function capture_order( $order_id ) {
+	public function capture_order( $order_id, $order_data ) {
 		$query_args = [];
-		$body       = [];
+
+		// Todo @rafsuntaskin handle $data validation to avoid errors.
+		$body       = [
+			'payerID' => $order_data['payerID'],
+		];
```

Comment:
Where is this data coming from?

Reply:
This data is coming from PayPal and is available within the `data` argument of `handleApprove` method: https://github.com/the-events-calendar/event-tickets/blob/e29c36811a50456c2c91c88e45ff3508af9d1edd/src/resources/js/commerce/gateway/paypal/checkout.js#L218

Reply:
I have added this data within the Order capture API call: https://github.com/the-events-calendar/event-tickets/blob/e29c36811a50456c2c91c88e45ff3508af9d1edd/src/resources/js/commerce/gateway/paypal/checkout.js#L231